research/mobilenet/preprocessing_slim.py

```diff
@@ -0,0 +1,324 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Quick question about this preporcessing, is it mainly for loading tf1 checkpoints?